### DISCUSSION TITLE
请问为什么我的PR交流着就没有回信了？

### DISCUSSION THREAD

**Discussion by wfan-os** [CFPAOrg/Minecraft-Mod-Language-Package#3121] [Q&A] [answered]
我提交了以下：
![Screenshot 2023-03-18 115345](https://user-images.githubusercontent.com/72018068/226083780-6a3d11a3-2585-4bbd-bc13-1027a9bebbd0.png)
这些的模组的翻译PR，可很多审核人员在和我交流之后（甚至很多是交流着交流着）就突然了无音讯了，就没有再找过我了。
能请问这是怎么回事吗？而且我发现不止我这样，我之前的很多模组PR也面临着这样的情况，就这么卡住了。
这接近200个PR怎么办啊，我的PR不会一直卡在这里直到被人遗忘吧。。。

**Comment by ChuijkYahus** [comment, marked as answer]
最近审核人手一直处于不足的状态，并且审核也都不是全职工作，不能及时进行讨论和给予回复十分抱歉。
请放心，所有PR最后都是会审核的，但也请按照审核的要求开游戏进行自查。

**Comment by luoxueyasha** [comment]
> 若有不明白的地方，可前往 QQ 群（630943368，较为活跃）或 [Discord](https://discord.com/invite/SGve5Fn) 提问。

摘录自 [CONTRIBUTING](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/main/CONTRIBUTING.md)
欢迎来QQ群进行进一步讨论
